neuralmonkey/decoding_function.py

```diff
@@ -1,59 +1,93 @@
 import tensorflow as tf
 
-def attention_decoder(decoder_inputs, initial_state, attention_objects,
-                      embedding_size, cell, output_size=None,
-                      loop_function=None, dtype=tf.float32, scope=None):
+from neuralmonkey.logging import debug
 
-    if output_size is None:
-        output_size = cell.output_size
+def attention_decoder(decoder_inputs, initial_state, attention_objects,
```

Comment:
Zahrnul bych to do refactoru `decoding_function` v samostatný issue nebo v rámci #9